client/my-sites/checkout/src/lib/leave-checkout.ts

```diff
@@ -8,11 +8,12 @@ import {
 	getSignupCompleteFlowName,
 	retrieveSignupDestination,
 } from 'calypso/signup/storageUtils';
+import { sendMessageToOpener } from './popup';
 
 const debug = debugFactory( 'calypso:leave-checkout' );
 
 export const leaveCheckout = ( {
-	siteSlug,
+	siteSlug = '',
```

Comment:
Same here: what if we do not change this, but instead provide a fallback when we call `sendMessageToOpener`? Otherwise we risk changing behavior for other things just to cover this one case.

Reply:
Sounds good 👍